templates/IAM/github-oidc-provider-access.njk

```diff
@@ -0,0 +1,115 @@
+{# Allow Github to access AWS without giving github AWS credentials               #}
+{# Must pass in the following templating parameters                               #}
+{#  Parameters:                                                                   #}
+{#    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ] #}
+{#    ManagedPolicyArns:                                                          #}
+{#      - "arn:aws:iam::aws:policy/AdministratorAccess"                           #}
+
+{# Make template compatible with both sceptre and org-formation     #}
+{% if sceptre_user_data.Repositories is defined %}
+    {% set Repositories = sceptre_user_data.Repositories %}
+{% endif %}
```

Comment:
are we going to support this compatibility?  or do we only want this for sceptre?  if we want to support this then it needs to be tested and documented